PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -2414,9 +2414,9 @@ void ScintillaEditView::convertSelectedTextTo(bool Case)
 		for (int i = 0 ; i < nbChar ; ++i)
 		{
 			if (Case == UPPERCASE)
-				selectedStrW[i] = (WCHAR)::CharUpperW((LPWSTR)selectedStrW[i]);
+				selectedStrW[i] = (WCHAR)(UINT_PTR)::CharUpperW((LPWSTR)selectedStrW[i]);
 			else
-				selectedStrW[i] = (WCHAR)::CharLowerW((LPWSTR)selectedStrW[i]);
+				selectedStrW[i] = (WCHAR)(UINT_PTR)::CharLowerW((LPWSTR)selectedStrW[i]);

```

Comment:
> Your patch is a good one, you should propose it as a pull request and in your pull request use better replacement fro this API :)

Well, however much I liked that patch (it wasn't perfect, but I _did_ like it), it was just far too big for review, and thus never had a chance at merging :)

Besides, I can't even get [single-line changes merged](https://github.com/donho/notepad-plus-plus/pull/31), let alone [fixes](https://github.com/donho/notepad-plus-plus/pull/32), or really, [much of anything else](https://github.com/donho/notepad-plus-plus/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Aariccio+).